package.json

```diff
@@ -56,7 +56,6 @@
     "moment": "2.19.4",
     "moment-duration-format": "2.2.2",
     "postis": "2.2.0",
-    "prop-types": "15.6.0",
```

Comment:
I guess it's sad atlaskit uses proptypes so it's still in package-lock.json.

Reply:
Indeed.